From 7fe8b271815072b3ad90007cb1a58268f0fefdaf Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 21 Jan 2014 10:44:12 +0800
Subject: [PATCH 0419/1543] ENGR00295892-2: ARM: dts: imx6qdl-sabresd: add
 retain-state-suspended property in dts

commit d73df441bc2f6fcfdc7868ce6f902b058d68931b from
git://git.freescale.com/imx/linux-2.6-imx.git

Add property "retain-state-suspended" in dts.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit aff480769c26ff7f64059f9631b10e8177c4791b)
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 708fe7c..ee07ab9 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -39,6 +39,7 @@
 		charger-led {
 			gpios = <&gpio1 2 0>;
 			linux,default-trigger = "max8903-charger-charging";
+			retain-state-suspended;
 		};
 	};
 
-- 
1.7.5.4

